evalml/model_understanding/graphs.py

```diff
@@ -516,6 +516,8 @@ def partial_dependence(pipeline, X, features, grid_resolution=100):
             If features is an int, it must be the index of the feature to use.
             If features is a string, it must be a valid column name in X.
             If features is a tuple of int/strings, it must contain valid column integers/names in X.
+        percentiles (tuple[float]): The lower and upper percentile used to create the extreme values for the grid.
+            Must be in [0, 1]. Defaults to (0.05, 0.95).
         grid_resolution (int): Number of samples of feature(s) for partial dependence plot.  If this value
             is less than the maximum number of categories present in categorical data within X, it will be
             set to the max number of categories + 1.
```

Comment:
Could you add the default value for grid_resolution in for me?  I think I failed to do so ages ago.  My bad.